psignifit/getConfRegion.py

```diff
@@ -48,12 +48,111 @@
 # return( list( indices = which( probMassVec >= HDIheight ) ,
 # mass = HDImass , height = HDIheight ) )
 # }
+from typing import Sequence
 
 import numpy as np
+from scipy import stats
 
 from .marginalize import marginalize
 
 
+def confidence_intervals(probability_mass: np.ndarray, grid_values: np.ndarray,
```

Comment:
`probability_mass` is now hidden within the `_fit_parameters` function. How do we get it here?

  - we are actually calculating the confidence intervals already within the function `psygnifit`, where we have access to the `probability_mass`
  - should we attach `probability_mass` to `results`? But then it is a huge array that we are carrying arround
  - or, make the storing of the `probability_mass` optional, so that you can only calculate the confidence intervals for different parameters if you have optionally stored the `probability_mass`